tensorcircuit/templates/conversions.py

```diff
@@ -36,3 +38,201 @@ def get_ps(qo: Any, n: int) -> Tuple[Tensor, Tensor]:
         res.append(res_t)
         wts.append(w)
     return np.array(res), np.array(wts)
+
+
+def QUBO_to_Ising(Q: List[list]) -> Tuple[List[list], list, float]:
+    """
+    Cnvert the Q matrix into a the indication of pauli terms, the corresponding weights, and the offset.
```

Comment:
typo: Cnvert
typo: into a the indication of 
typo: Pauli should be capitalized